README.md

```diff
@@ -93,34 +121,50 @@ To start developing your Worker, run `cd flow-workers && npm start`
 To publish your Worker to the Internet, run `npm run deploy`
 ```
 
-Now `cd flow-workers`.
+3. Now, `cd flow-workers`. You'll work with both Flow and Cloudflare in this directory.
 
-### Create the Flow derivation
+## Create the Flow derivation
 
-- Authenticate `flowctl` by visiting (https://dashboard.estuary.dev/admin/api), copying the access token, and pasting that token in your shell:
-  - `flowctl auth token --token <your-token-here>`
-- Create a draft: `flowctl draft create` will output the ID of your draft. `flowctl` will also persist this ID and use it on subsequent invocations, so you don't need to remember it
-- Add the source collection to your: `flowctl catalog draft --name estuary/public/examples/flow-github/workflow_runs`. Substitute your own collection name, if you're following along.
-- Run `flowctl draft develop` to pull down the specification into your local directory
+1.  Authenticate `flowctl` by copying an access token from [the web app](https://dashboard.estuary.dev/admin/api) and pasting it in your shell:
```

Comment:
Just a heads up that this will likely change a bit once estuary/flow#792 is merged.